ui/k-info-panel/switch_info.kytos

```diff
@@ -55,13 +63,42 @@
        this.get_flows()
      },
      get_flows() {
+       this.nextState = this.metadata.enabled == 'true'? 'Disable': 'Enable'
+       console.log(this.nextState)
```

Comment:
@Alopalao could you remove these temporary console.log statements?

Reply:
I deleted it.

Reply:
It is now deleted. Commit [4e91ab3ee60976a1d6fabfb1cb041793c253b9ed](https://github.com/kytos-ng/topology/pull/70/commits/4e91ab3ee60976a1d6fabfb1cb041793c253b9ed)